client/src/components/utils/LoginForm/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Link } from "react-router-dom";
+import { Form, Icon, Input, Checkbox } from "antd";
+import PropTypes from "prop-types";
+
+import Button from "../Button";
+import "./style.css";
+
+class LoginForm extends Component {
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFields((err, values) => {
+      if (!err)
+        fetch("/login", { method: "POST", body: JSON.stringify(values) });
```

Comment:
@ahmedisam99 Once you're ready to add a redirect here, after a successful login, you can use the `withRouter` HOC to wrap this component's export, and then use this.props.history.push here (without having to pass in a history prop yourself).
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md